src/backend/storeManagers/hyperplay/games.ts

```diff
@@ -1202,3 +1217,18 @@ export async function syncSaves(
 export async function forceUninstall(appName: string): Promise<void> {
   await uninstall({ appName, shouldRemovePrefix: false })
 }
+
+function writeManifestFile(installedInfo: Partial<InstalledInfo>) {
+  const manifestPath = path.join(
+    path.dirname(installedInfo.install_path!),
+    'manifest.json'
+  )
+  const manifest = JSON.stringify(installedInfo, null, 2)
+
+  // delete manifest if exists
+  if (existsSync(manifestPath)) {
```

Comment:
If this is to avoid override errors there's a flag we can use for the `writeFileSync`

```js
writeFileSync('path', 'data', {encoding: 'utf8', flag: 'w'})
```

Reply:
cool, will update.